### ISSUE TITLE
duplicated 5m contour

### ISSUE THREAD

**Issue by quantenschaum** [wellenvogel/ochartsng#9]
![image](https://github.com/wellenvogel/ochartsng/assets/3985599/47da8115-a0da-4c34-8e7c-e5c22163db09)

**Comment by wellenvogel** [comment]
Could you describe which chart this is and what your depth settings are?
Background:
Safety depth should have thicker lines - so I need to find out if this is really a duplicated render - or some strange thick line rendering issue.

**Comment by quantenschaum** [comment]
example is west of Hiddensee, but it is like this everywhere. the safety contour seems be duplicated. I can live with that, but it looks irritating.

**Comment by wellenvogel** [comment]
yes, can reproduce that. Seems like a thick line rendering issue.
Will have a closer look at it.

**Comment by wellenvogel** [comment]
Currently the algorithm does not really support it (and it would become slower).
Anyway I'm just searching for some better algorithm for the thick line drawing - but for now this is maybe not the highest priority.
